src/actuator.c

```diff
@@ -79,7 +112,7 @@ static int momentary_process(cc_actuator_t *actuator, cc_assignment_t *assignmen
             }
 
             // toggle mode
-            else if (assignment->mode & CC_MODE_TOGGLE)
```

Comment:
this must be a typo..? toggle|trigger often go together, this change would make the trigger flag useless

Reply:
I don't entirely understand why this is raised as an issue. The trigger and toggle functionality are performing two different actions and are both needed in the momentary process.

Reply:
because the bitmask will have `CC_MODE_TOGGLE` and `CC_MODE_TRIGGER` at the same time (so both conditions evaluate as true).
you invalidate the previous check by modifying the `else if` to an `if`.